issues in 2.9.1 fixed and UI modified

SankariSM opened the pull request:
Attempted to load class issue - fixed,
notices in order import - fixed,
issues in order detail - fixed,
Module configuration - UI modified

SankariSM committed:
Attempted to load class issue - fixed,
notices in order import - fixed,
issues in order detail - fixed,
Module configuration - UI modified

SankariSM committed:
line space added

FabienSerny commented:
Hi,

The only thing that is "blocking" me to merge this PR is the fact that you use "table" "tr" "td" instead of div and bootstrap classes. Are you sure you want me to merge this anyway ?

SankariSM committed:
width removed added bootstrap class for tables

SankariSM commented:
> Hi,
> 
> The only thing that is "blocking" me to merge this PR is the fact that you use "table" "tr" "td" instead of div and bootstrap classes. Are you sure you want me to merge this anyway ?

Usually for listings i use table(which is easier to add styles). I had used the table along with the bootstrap class(table) to make the hover effect. 
Now removed width and replaced with bootstrap class. Is this ok or replace the old code?
